api/src/test/java/org/openmrs/module/attachments/obs/TestHelper.java

```diff
@@ -114,9 +114,9 @@ public void init() throws IOException {
 		
 		WebConstants.CONTEXT_PATH = "openmrs";
 		
-		context.getAdministrationService()
-		        .saveGlobalProperty(new GlobalProperty(AttachmentsConstants.GP_CONCEPT_COMPLEX_UUID_LIST,
-		                "[\"7cac8397-53cd-4f00-a6fe-028e8d743f8e\",\"42ed45fd-f3f6-44b6-bfc2-8bde1bb41e00\"]"));
+		context.getAdministrationService().saveGlobalProperty(new GlobalProperty(
+		        AttachmentsConstants.GP_CONCEPT_COMPLEX_UUID_LIST,
+		        "[\"7cac8397-53cd-4f00-a6fe-028e8d743f8e\",\"42ed45fd-f3f6-44b6-bfc2-8bde1bb41e00\",\"32d3611a-6699-4d52-823f-b4b788bac3e3\"]"));
```

Comment:
This specific setup should only belong to the one test that aserts the `APIException`, we don't want the misconfiguration to affect all tests.

Reply:
@ridmal the above comment has not been addressed yet, could you look into it?